docs/recipes/WritingTests.md

```diff
@@ -304,7 +304,7 @@ import Header from '../../components/Header'
 
 function setup() {
   const props = {
-    addTodo: expect.createSpy()
+    addTodo: jest.fn(() => {})
```

Comment:
@Florian-R indeed, according to the [documentation](https://facebook.github.io/jest/docs/api.html#jest-fn-implementation) that should work.

But weirdly, when I run jest with a no-args [`jest.fn()` in applyMiddleware.spec.js](https://github.com/reactjs/redux/pull/1951/commits/927bf45f1c7666e42ae8c4c390cc726a75cab903#diff-0c67d0d5b7357141efeb1d7e87115773R15) I get the following error which, to me, seems totally unrelated:
```
 FAIL  test/applyMiddleware.spec.js
  ● Test suite failed to run

    /Users/heinrichfilter/dev/src/redux/test/applyMiddleware.spec.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import { createStore, applyMiddleware } from '../src/index';
                                                                                             ^^^^^^
    SyntaxError: Unexpected token import

      at transformAndBuildScript (../../../.nvm/v6.5.0/lib/node_modules/jest/node_modules/jest-runtime/build/transform.js:284:10)
```

I'm a bit confused by it and thought I'd leave it in until someone can shed some light on it or I figure it out. Can someone maybe run the tests and see if they get the same error?